streamer/src/nonblocking/quic.rs

```diff
@@ -1228,16 +1317,43 @@ pub mod test {
         SocketAddr,
         Arc<StreamStats>,
     ) {
-        let s = UdpSocket::bind("127.0.0.1:0").unwrap();
+        let sockets = {
+            #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+            {
+                use std::{
+                    os::fd::{FromRawFd, IntoRawFd},
+                    str::FromStr as _,
+                };
+                (0..10)
+                    .map(|_| {
+                        let sock = socket2::Socket::new(
+                            socket2::Domain::IPV4,
+                            socket2::Type::DGRAM,
+                            Some(socket2::Protocol::UDP),
+                        )
+                        .unwrap();
+                        sock.set_reuse_port(true).unwrap();
```

Comment:
This is failing in CI 

    --> streamer/src/nonblocking/quic.rs:1335:30
     |
1335 |                         sock.set_reuse_port(true).unwrap();
     |                              ^^^^^^^^^^^^^^ method not found in `Socket`

Any idea? The socket2 version is 0.5.4 which has the function for socket2::Socket struct as I checked. @t-nelson

Reply:
hrm... yeah i don't see any changes in this regard wrt either `socket2` nor how we're declaring the dependency in the monorepo.  maybe try adding `features = ["all"]` to the `socket2` entry in `Cargo.toml`?

Reply:
yeah. it worked on my local build. I pushed it.